recipes/sam2pairwise/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "sam2pairwise" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin="x") }}
+
+source:
+  url: https://github.com/mlafave/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 899d3db071cc248870bbedbdb39784de4c54c9a7888bc408022e809ace47ec0e
+  patches:
+    - patch.txt
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+  run:
+    - make
```

Comment:
why is make needed here?

Reply:
Good point! Just forgot to fully remove the run section (since I believe there are no run dependencies, since it's just a single compiled executable). Removed it.